app/Models/RuleAction.php

```diff
@@ -93,4 +94,10 @@ protected function ruleId(): Attribute
             get: static fn ($value) => (int)$value,
         );
     }
+
+    public function getValue(array $journal)
```

Comment:
Will need a return value

Reply:
What happens when the action value is NOT an expression? Not sure of the method name either, perhaps something like "getExpressionValue"?

Reply:
The `ActionExpression` class just ignores any value that is not prefixed with an `=` sign, so the call to `evaluate` will just return the value as passed in. So `getValue` just returns the `action_value` if it's not an expression, or evaluates it and returns the result if it is an expression.